openjdk.test.jck/docs/README.md

```diff
@@ -0,0 +1,298 @@
+Resetting the /bin/sh symbolic link to bash works, but is obviously not ideal.* 
+
+<a name="setup"></a>
+## Setup
+The test.jck automation requires that the JCK has first been configured for execution
+(unzipped and the native ('C') tests built for the platform under test), and placed
+beneath a root directory with jck/&lt;jck-version&gt; subdirectories.
+
+For example, for JCK release jck8b, this would be the structure required:
+```
+<JCK_ROOT>/jck/jck8b/excludes
+<JCK_ROOT>/jck/jck8b/JCK-compiler-8b
+<JCK_ROOT>/jck/jck8b/JCK-devtools-8b
+<JCK_ROOT>/jck/jck8b/JCK-runtime-8b
+<JCK_ROOT>/jck/jck8b/natives
+```
+The default location for &lt;JCK_ROOT&gt; is the -systemtest-prereqs STF argument value
+(which may also be specified via the PREREQS_ROOT) makefile value. If a user has opted for the locations
+recommended in the instructions for the test repositories, this would be the directory structure for the
+git repositories and the test and prereqs (including the JCK):
+```
+/home/user/git/stf
+/home/user/git/stf/........                       <- stf repository files
+/home/user/git/openjdk-systemtests
+/home/user/git/openjdk-systemtests/........       <- openjdk-systemtest repository files
+/home/user/systemtest_prereqs
+/home/user/systemtest_prereqs/junit-4.12/........ <- junit files
+/home/user/systemtest_prereqs/jck
+/home/user/systemtest_prereqs/jck/jck8b           <- Java 8 JCK (jck8b) files
+/home/user/systemtest_prereqs/.........           <- other prereq directories
+```
+The JCK tests also require certain configuration files.  Some of these are included in the openjdk-systemtest
+repository. For example, for the jck8b JCK release:
+```
+test.jck/config/jck8b/compiler.jti
+test.jck/config/jck8b/devtools.jti
+test.jck/config/jck8b/runtime.jti
+```
+Two additional configuration files are required which must be set up according to the test machine environment.
+There are templates for these files included in the openjdk-systemtest repository
+```
+test.jck/config/default/jcktest.properties
+test.jck/config/default/krb5.conf
+```
+The files must be copied to a config/&lt;config-name&gt; subdirectory beneath a systemtest-prereqs directory
+and the values updated to reflect the test machine setup.  If a value other than 'default' is used for
+config-name then config-name will need to be supplied to the test at execution time.
+
+The simplest way to set up these files is to place them alongside the 'ready to run' JCK itself - for example:
+```
+<JCK_ROOT>/jck/jck8b/config/default/jcktest.properties
+<JCK_ROOT>/jck/jck8b/config/default/krb5.conf
+```
+The correct values are set according to the Kerberos and Http server setup which the
+JCK tests are to access.  The JCK documentation describes how to set up those services.
+
+The natives can be built by executing the GNU makefile test.jck/src/native/makefile as follows:
+```
+make PLATFORM=xxxx SRCDIR=<path-to-jck-root>/JCK-runtime-8b OUTPUT=<path-to-jck-root</natives
```

Comment:
"<" should be ">" at the end of this before /natives

Reply:
Fixed